src/main/java/redis/clients/jedis/search/SearchResult.java

```diff
@@ -33,12 +33,16 @@ public static class SearchResultBuilder extends Builder<SearchResult> {
     private final boolean hasContent;
     private final boolean hasScores;
     private final boolean hasPayloads;
+    private final boolean hasSortKeys;
+    private final boolean hasExplainScore;
     private final boolean decode;
 
-    public SearchResultBuilder(boolean hasContent, boolean hasScores, boolean hasPayloads, boolean decode) {
+    public SearchResultBuilder(boolean hasContent, boolean hasScores, boolean hasPayloads, boolean hasSortKeys, boolean hasExplainScore, boolean decode) {
```

Comment:
Breaking change. We can't _modify_ this now.

Reply:
Ok. Removed the changes related to sortKeys & explainScore from this PR.
Will create a separate PR for those to be included in 5.0.0 release.